install.sh

```diff
@@ -94,14 +94,27 @@ nvm_download() {
 install_nvm_from_git() {
   local INSTALL_DIR
   INSTALL_DIR="$(nvm_install_dir)"
+  local NVM_VERSION
+  NVM_VERSION="${NVM_INSTALL_VERSION:-$(nvm_latest_version)}"
+  local IS_TAG
+  IS_TAG=$(command printf '%s\n' "$NVM_VERSION" | GREP_OPTIONS='' command grep -c -E '^v[0-9]+.[0-9]+.[0-9]+$')
```

Comment:
When I tried to copy the nvm_grep function from nvm.sh to use it in install.sh, the test intall_script/nvm_install_with_node_version was failing, I did not figure out why.
Is it not weird ?

Reply:
Not sure; let's go back to the copied `nvm_grep` function and i'll debug the error.

However, i'm not sure this is the best way to detect the tag - is there no git command that could tell us that?

Reply:
git cannot tell much as we clone with depth 1, so we dont have much information.
I could add git to fetch the tag though ? That would be indeed more relevant and work if the format of the release evolve.

Reply:
Ah yes it can :D

Reply:
ah, true, probably not.

Reply:
Does it look better now ?
Also I do not use grep anymore but I made another PR to show you the problem.